### DISCUSSION TITLE
WebGL向けビルドのエラーについて

### DISCUSSION THREAD

**Discussion by takenokohal** [pixiv/vroid-sdk-developers#26] [Q&A]
現在のSDKでWebGLにビルドをし、モデルのロードをしようとするとCORSに引っかかってエラーになってしまいます。
また、ChromeにてCORSを無視して起動しても、次はエンコードエラー？に引っかかってロードができませんでした。

WebGLビルドでVRoidSDKを使う方法はあるでしょうか？
また、正式にはWebGL向けサポートはしていないとのことですが、今後対応する予定はあるでしょうか？

**Comment by Haresaku** [comment]
現在、VRoid SDKはWebGL向けのビルド をサポートしておりません。お手数ですが別のプラットフォーム向けのビルドでお試しください。
